src/main/java/me/magnet/consultant/Consultant.java

```diff
@@ -356,6 +367,9 @@ public Consultant build() {
 			instanceName = Optional.ofNullable(instanceName)
 					.orElse(Optional.ofNullable(fromEnvironment("SERVICE_INSTANCE"))
 							.orElse(UUID.randomUUID().toString()));
+			Arrays.stream(Strings.nullToEmpty(fromEnvironment("SERVICE_TAGS")).split(","))
+					.filter((t) -> t != null && !t.isEmpty())
```

Comment:
Bit of a bummer Java does not allow method reference negations eg `filter(!Strings::isNullOrEmpty)` so we'll do it the older fashioned way